cypress/page-objects/shopPage.js

```diff
@@ -1,8 +1,8 @@
 import BasePage from './basePage';
 
 class ShopPage extends BasePage {
-  addProductToCart(pickedItem, quantity = 1) {
-    cy.get(`div[name="${pickedItem.name}"]`).parent().find('button').invoke('show')
+  addProductToCart(product, quantity = 1) {
```

Comment:
rework it to take array of products, but do not touch quantity in args - it looks good
it doesn't mean that `forEach` looks bad in test, but we just can get rid of that iterator inside of the test step and add it right here

Reply:
I haven't done this yet since it will require changes in other places, and this won't help to get rid of forEach, bc it will appear in other places. Let me explain it on Monday just to not type a lot of text here.

Reply:
Finally done <3